### PR TITLE
Changed model types for AmericanPayroll

### PR DESCRIPTION
**Pull request by thevalyreangroup**
Changed from Int32 to String for both AccountNumber and RoutingNumber.  And Int32 is not large enough to hold most American bank account numbers and the XERO API is expecting a string.

### CODE REVIEW

File: Xero.Api/Payroll/America/Request/PayItemsRequest.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        public void Add(PayItems value)
+        {
+            PayItems = value;
+        }
+
+        public void AddRange(IEnumerable<PayItems> value)
+        {
+            PayItems = value.FirstOrDefault();
+        }*/
+    }
```
Comment (MJMortimer) [inline Xero.Api/Payroll/America/Request/PayItemsRequest.cs]: Hey @thevalyreangroup, Can you please clean this class up?

Comment (thevalyreangroup) [inline Xero.Api/Payroll/America/Request/PayItemsRequest.cs]: @MJMortimer can you please clarify what you mean?

Comment (MJMortimer) [inline Xero.Api/Payroll/America/Request/PayItemsRequest.cs]: All of the contents of the file have been commented out. If the inner code of the class is unneeded the inner code should just be removed

Comment (thevalyreangroup) [inline Xero.Api/Payroll/America/Request/PayItemsRequest.cs]: @MJMortimer, Got ya. I will fix tomorrow.

Comment (MJMortimer) [reply Xero.Api/Payroll/America/Request/PayItemsRequest.cs]: Have you been able to make these changes yet @thevalyreangroup?

Comment (thevalyreangroup) [reply Xero.Api/Payroll/America/Request/PayItemsRequest.cs]: @MJMortimer Fixed.

**Pull request merged**